src/openapi/schemas/typename.yaml

```diff
@@ -27,6 +27,7 @@ components:
           Community,
           CommunityDeleteActionUnavailable,
           CommunityJoinAction,
+          CommunityJoinActionUnavailable,
```

Comment:
Can you tell me the endpoint that contains the response that causes `CommunityJoinActionUnavailable` and the Twitter URL?

I want to see the detailed model response.

Reply:
Sure! I actually saw it happen from a few endpoints, like HomeTimeline and TweetDetail. I think this may be injected into HomeTimeline when you follow an account that makes a community post in a community that requires approval to join. Here's an example tweet and the relevant API response. I was logged into an account when this API response was received. 

https://x.com/cyb_detective/status/1853879226987901408

```
                                },
                                "join_action_result": {
                                  "__typename": "CommunityJoinActionUnavailable",
                                  "reason": "ViewerRequestRequired",
                                  "message": "You must request to join this Community."
                                },
```